### DISCUSSION TITLE
Should we remove the Blog top level link?

### DISCUSSION THREAD

**Discussion by pauby** [chocolatey/boxstarter.org#3] [Ideas]
The blog link on the boxstarter.org website points to boxstarter.org/blog. We've never added anything in there. Should we remove this entirely OR point it to the boxstarter tag (assuming we have one) on the Chocolatey blog?

**Comment by gep13** [comment]
I actually don't know if we even have control of this.  The way I believe that this currently works is that this is a curated list of Matt's blog posts within a FeedBurner feed, which is then displayed on the Blog page.

I would vote for removing this.  Although, I would recommend that we come to a decision point about "where" the new site is going to be, i.e. within a Statiq site (assuming that the ClickOnce installer works there) or elsewhere, that way we don't have to duplicate the changes.

**Comment by pauby** [comment]
Under the blog link I don't see anything:

![image](https://user-images.githubusercontent.com/12760779/147963917-13a82d27-788c-4636-880e-90b225e3a3d2.png)

I think we should use Statiq (as the team has experience in that and we're using it elsewhere) and host it here (or `chocolatey-community`).

**Reply by gep13** [reply]
I suspect you have something in the browser blocking it from appearing:

![image](https://user-images.githubusercontent.com/1271146/147964632-aef9e4aa-65e1-4eee-8c22-12d7e0e06f8a.png)

**Reply by pauby** [reply]
Yeah. Building the site from the Docker container it shows exactly the above. Latest post is 2014 so I think my suggestion may still be valid 😄

**Comment by pauby** [comment]
I think we should have a way of showing Matt's posts AND the ones from the [Chocolatey blog](https://blog.chocolatey.org) that have the 'boxstarter' tag?

**Reply by gep13** [reply]
I am not sure exactly how this would work in practice. We might be able to do something in Statiq, to bring in the relevant posts from each source at the point of generating the site, or we would need to look for some form of RSS syndicator to put together the posts that we want from each location, to then allow that to flow into the site.  Not entirely sure how to do that though.
